final-project/scripts/main.js

```diff
@@ -1,10 +1,110 @@
-import "../styles/main.scss";
-
-// testing that `const` and arrow functions transpile correctly
-const before = "Hello -world!";
-const after = before
-  .split("")
-  .filter(char => char !== "-")
-  .join("");
-console.log({ before });
-console.log({ after });
+import "../styles/main.scss";
+import Axios from "axios";
+
+function showPage() {
+  document.getElementById("body-layout").classList.toggle("layout-wrapper--loading");
+  document.getElementById("page-loader").style.display = "none";
+}
+
+window.onload = function () {
+  //Wait just to simulate a low connection and a delay
+  setTimeout(showPage, 3000);
+
+  //Retrieve blog posts
+  const url = 'https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog';
+  Axios
+    .get(url)
+    .then(data => {
+      const articles = data.data.articles;
+      for (let i = 0; i < articles.length; i++) {
+        const article = articles[i];
+
+        // Add articles
+        $('#main-blog-articles')

```

Comment:
Just as a recommendation, when selecting nodes with js is better to do it with selectors that use a prefix, ie: .js-my-class-selector
Although is not that bad to have used an id as long as you don't use that id for styling purposes